rollup.config.babel.js

```diff
@@ -38,3 +33,15 @@ export default {
     commonjs()
   ]
 }
+
+if (!process.env.NO_MODULE) {
+  config.output.push({
```

Comment:
I'm not sure what's going on or if I confused myself, but I was getting errors when the es/module version was available in the build directory.

I noted that the published 0.0.9 build/  only contained the cjs bundle